app/models/worldwide_organisation.rb

```diff
@@ -145,4 +145,8 @@ def republish_worldwide_offices
 
     offices.each { |office| Whitehall::PublishingApi.republish_async(office) }
   end
+
+  def search_index
+    super.merge("world_locations" => world_locations.map(&:slug))
```

Comment:
What happens when a world location has its slug changed? Does everything search-related get updated automatically?

Reply:
Unfortunately at some point in the past, slug was chosen as the way of linking to world locations in Search API.  This means some manual work is required if a location's slug changes.  We have that [documented](https://docs.publishing.service.gov.uk/manual/rename-a-country.html#6-remove-duplicate-search-results) as a step in the "Rename a country" playbook.